plutus-tx/src/PlutusTx/Builtins.hs

```diff
@@ -591,3 +593,17 @@ bls12_381_mulMlResult = BI.bls12_381_mulMlResult
 {-# INLINABLE bls12_381_finalVerify #-}
 bls12_381_finalVerify :: BuiltinBLS12_381_MlResult -> BuiltinBLS12_381_MlResult -> Bool
 bls12_381_finalVerify a b = fromBuiltin (BI.bls12_381_finalVerify a b)
+
+-- Conversions
+
+-- | Convert a 'BuiltinInteger' into a 'BuiltinByteString', as described in
+-- [CIP-0087](https://github.com/mlabs-haskell/CIPs/blob/koz/to-from-bytestring/CIP-0087/CIP-0087.md#builtinintegertobytestring).
```

Comment:
The idea is that they all should be the same: I was working on this concurrently, so that's probably why it's inconsistent. The goal is very much to link to the officially-accepted version when it's time. For now I'll make sure they all reference the PR.